nix-tools-release.nix

```diff
@@ -0,0 +1,69 @@
+commonLib: # the iohk-nix commonLib
+{ packages ? []
+, required-name ? "required"
+, required-targets ? (jobsets: [])
+, config ? {}
+, rev ? "abcdef"
+, package-set-path # usually import ./. {}
+}:
+{ system ? builtins.currentSystem
+, pkgs ? commonLib.getPkgs { inherit system config; }
+
+, scrubJobs ? true
+, supportedSystems ? [ "x86_64-linux" "x86_64-darwin" ]
+, nixpkgsArgs ? {
+    config = config // { allowUnfree = false; inHydra = true; };
+  }
+}:
+with (import (commonLib.nixpkgs + "/pkgs/top-level/release-lib.nix") {
+  inherit supportedSystems scrubJobs nixpkgsArgs;
+  packageSet = import package-set-path;
+});
+with pkgs.lib;
+let
+
+  traceId = x: builtins.trace (builtins.deepSeq x x) x;
+
+  packageSet = import package-set-path {};
```

Comment:
should we not be passing some args here?

Reply:
We could try to artificially construct arguments, but as we are only interested in the nix-tools attribute, to generate the proper inputs for the mapTestOn and mapTestOnCross functions, an empty set of arguments should do.